src/NzbDrone.Core/Datastore/Migration/182_add_bypass_to_delay_profile.cs

```diff
@@ -0,0 +1,15 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(182)]
+    public class add_custom_format_score_bypass_to_delay_profile : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("DelayProfiles").AddColumn("BypassIfAboveCustomFormatScore").AsBoolean().Nullable();
```

Comment:
Should we make this boolean non-nullable with default false, similar to the bypass quality setting that was added

```suggestion
            Alter.Table("DelayProfiles").AddColumn("BypassIfAboveCustomFormatScore").AsBoolean().WithDefaultValue(false);
```